scheduler/system_sched.go

```diff
@@ -18,6 +18,7 @@ const (
 
 // SystemScheduler is used for 'system' jobs. This scheduler is
 // designed for services that should be run on every client.
+// One for each job, containing an allocation for each node
```

Comment:
This is true for all schedulers I believe.  Each scheduler instance is created to schedule a single eval.